src/components/qas/questions/qaMultiSelect/qaMultiSelect.module.scss

```diff
@@ -0,0 +1,69 @@
+:global {
+  @include react-md-menus-media;
+  @include react-md-selection-controls;
+
```

Comment:
I think it behooves us to use this global trick as exclusively as we can.  And in my testing wrapping only these includes in global still yielded the styling we'd expect.

Reply:
Not sure if this is an action item or not.